tests/test_data_fetch.py

```diff
@@ -59,3 +59,11 @@ def test_supported_url():
 
 def test_unsupported_url():
     assert not data_fetch.can_fetch_from("aaaaa")
+
+
+def test_fetch_local(tmp_path):
+    source_path = pathlib.Path(__file__).absolute()
+    dest_path = tmp_path / "dest"
+    data_fetch.fetch_data("file://" + str(source_path), dest_path)
```

Comment:
So if I am an user when I run the installation where do I put the files or how do I pass files to Anaconda?

Reply:
You can use this only if those files are already there. I can imagine usage when the content filename is not guessed properly (e.g. RHEL8 vs RHV), or when the installation image contains the content as a specific location.